hypervisor/arch/x86/irq.c

```diff
@@ -369,8 +369,7 @@ void dispatch_exception(struct intr_excp_ctx *ctx)
 	cpu_dead();
 }
 
-#ifdef CONFIG_PARTITION_MODE
-void partition_mode_dispatch_interrupt(struct intr_excp_ctx *ctx)
+void dispatch_interrupt_lapic_pt(struct intr_excp_ctx *ctx)
```

Comment:
This dispatch function need be removed after changes done for partition mode, this is a TODO.

Reply:
Yes. It's a TODO